addons/opt_vehicles/cars/config.cpp

```diff
@@ -28,10 +28,67 @@ class CfgVehicles
 
 	class LandVehicle;
 
-	class Car;
+	class Car : LandVehicle
+	{
+		class ACE_SelfActions;
+	};
 
 	class Car_F : Car
 	{
+		TFAR_hasIntercom = 1;
```

Comment:
Du gibst das Intercom pauschal allen Fahrzeugen dieser Konfiguration. Also Ifrit, Hunter, Prowler, Qilin, Quadbike, SUV, off-roads, Quadbike und sogar UGVs. Ich denke zumindest für letztere ergibt das wenig Sinn.

Reply:
Vorschlag: Eine `class Car_F_Intercom : Car_F` anlegen, die die TFAR-Intercom-Einträge enthält. Und dann diese als Baseclass nur für jene Fzg-Typen nutzen, die Intercom haben sollten. Und falls es wegen komischer Vanilla-Vererbungen nicht geht, müssen die Einträge halt mehrfach rein.

Reply:
Wenn wirklich nur die Fahrzeuge haben sollen die und keine anderen haben dürfen ,dann kann ich da besser in die MRAP Basisclasse das kein machen. 
Aber warum ist das ein Problem wenn, das alle Rad Fahrzeuge hätten. 
Das macht doch überhaupt nix aus.

Reply:
InterCom auf Hunter/Irif/Strider beschränkt, damit du ruhig schlafen kannst Senshi

Reply:
Hast du Recht, das hatte ich dann wohl falsch im Kopf. Sorry, bin die Woche wie gesagt nur mobil unterwegs. :(

Reply:
Wieder zurücksetzen auf Car_F, sorry :( .

Reply:
wie auf Ursprung zu zurück gestellt.